backend/generator.js

```diff
@@ -10,10 +10,10 @@ exports.executeScript = function (socket, formData) {
     return false;
   }
   var email = formData.email;
-  var gitUrl = formData.gitUrl;
+  var gitUrl = "https://github.com/" + formData.username + "/" + formData.reponame + ".git";
```

Comment:
In the webUI, we are using username, reponame in some places and gitURL in others. using username, reponame seperately in the webUI would make it consistent.